lib/generators/module_component/module_component_generator.rb

```diff
@@ -18,21 +18,29 @@ def create_component
     # Take each passed in argument (e.g.) controller, serializer, etc
     # and create the corresponding files within the module for each arg
     path = "modules/#{file_name}/app"
-    methods.map(&:downcase).each do |method|
-      if COMPONENT_TYPES.include? method
-
-        template_name = method == 'model' ? "#{file_name}.rb" : "#{file_name}_#{method}.rb"
-        template "app/#{method.pluralize}/#{method}.rb.erb",
-                 File.join(path, method.pluralize.to_s, file_name, 'v0', template_name.to_s)
-
-        if method == 'service'
-          template "app/#{method.pluralize}/configuration.rb.erb",
-                   File.join(path, method.pluralize.to_s, file_name, 'v0', 'configuration.rb')
-        end
-      else
-        $stdout.puts "\n#{method} is not a known generator command."\
-          "Commands allowed are controller, model, serializer and service\n"
+    methods_hash = methods.to_h
```

Comment:
Isn't the methods arg already a hash?

Reply:
It is a hash to Thor at least, but I couldn't seem to pull anything out of it unless I added the .to_h